src/comestible_inv_pane.cpp

```diff
@@ -0,0 +1,672 @@
+        std::string key = comestible_inv_area::get_info(
+                              static_cast<comestible_inv_area_info::aim_location>( i ) ).minimapname;
+
+        comestible_inv_area *iter_square = get_square( i );
+
+        const char *bracket = iter_square->has_vehicle() ? "<>" : "[]";
+
+        bool is_single = cur_area->info.type != comestible_inv_area_info::AREA_TYPE_MULTI;
+        bool is_selected_by_user = cur_area->info.has_area( get_square( i )->info.id );
+        bool is_selected_by_item;
+        if( is_single ) {
+            is_selected_by_item = is_selected_by_user;
+        } else {
+            is_selected_by_item = get_square( i )->info.id == sel_area->info.id;
+        }
+        nc_color bcolor = c_red;
+        nc_color kcolor = c_red;
+        if( get_square( i )->is_valid() ) {
+            if( is_single ) {
+                if( is_selected_by_item ) {
+                    if( viewing_cargo ) {
+                        bcolor = c_light_blue;
+                    } else {
+                        bcolor = c_light_gray;
+                    }
+                    kcolor = c_white;
+                } else {
+                    bcolor = kcolor = c_dark_gray;
+                }
+            } else {
+                if( cur_area->info.id == get_square( i )->info.id ) {
+                    bcolor = c_light_gray;
+                    kcolor = c_white;
+                } else {
+                    if( is_selected_by_user ) {
+                        bcolor = c_light_gray;
+                    } else {
+                        bcolor = c_dark_gray;
+                    }
+
+                    if( is_selected_by_item ) {
+                        kcolor = c_light_gray;
+                    } else {
+                        kcolor = c_dark_gray;
+                    }
+                }
+            }
+        }
+
+        const int x = get_square( i )->info.hscreen.x + ofs;
+        const int y = get_square( i )->info.hscreen.y;
+        mvwprintz( window, point( x, y ), bcolor, "%c", bracket[0] );
+        wprintz( window, kcolor, "%s", viewing_cargo && is_selected_by_item && is_single &&
+                 cur_area->info.id != comestible_inv_area_info::AIM_DRAGGED ? "V" : key );
+        wprintz( window, bcolor, "%c", bracket[1] );
+    }
+    return get_square( comestible_inv_area_info::AIM_INVENTORY )->info.hscreen.y + ofs;
+}
+
+void comestible_inventory_pane_food::init( int items_per_page, catacurses::window w,
+        std::array<comestible_inv_area, comestible_inv_area_info::NUM_AIM_LOCATIONS> *s )
+{
+    columns = { COLUMN_CALORIES, COLUMN_QUENCH, COLUMN_JOY };
+    if( g->u.can_estimate_rot() ) {
+        columns.emplace_back( COLUMN_EXPIRES );
+    } else {
+        columns.emplace_back( COLUMN_SHELF_LIFE );
+    }
+    special_filter = []( const item & it ) {
+        const std::string n = it.get_category().name();
+        if( uistate.comestible_save.show_food && n != "FOOD" ) {
```

Comment:
This compares the *translated* category name to a fixed string. This won't work as intended if non-English used in the game. Same below.